runner/src/scenario/ceramic/mod.rs

```diff
@@ -130,28 +130,41 @@ async fn update_small_model(user: &mut GooseUser) -> TransactionResult {
             .unwrap()
     };
     let req = user.client.post(url).json(&req);
-    let req = GooseRequest::builder()
-        .method(GooseMethod::Post)
-        .set_request_builder(req)
-        .expect_status_code(200)
-        .build();
-    let resp = user.request(req).await?;
-    let resp: StreamsResponseOrError = resp.response?.json().await?;
-    resp.resolve("update_small_model").unwrap();
+    let mut goose = user
+        .request(
+            GooseRequest::builder()
+                .method(GooseMethod::Post)
+                .set_request_builder(req)
+                .expect_status_code(200)
+                .build(),
+        )
+        .await?;
+    let resp: StreamsResponseOrError = goose.response?.json().await?;
```

Comment:
Is the `?` at the end desired if we want the `goose_try` macro to resolve both `Ok` and `Err` results?

IIUC, we'll return early if there's an error and so the macro will only ever see successful results.

Reply:
This one is expected, since this would be the error if goose can't convert the response to json and the expected type.